be1-go/channel/lao/mod.go

```diff
@@ -219,6 +240,9 @@ func (c *Channel) Publish(publish method.Publish) error {
 	}
 
 	object, action, err := messagedata.GetObjectAndAction(jsonData)
+	if err != nil {
+		return xerrors.Errorf("impossible to get the action or the object: %v", err)
```

Comment:
"failed to" is mostly used, instead of "impossible to" (see above and elsewhere in the code)